### PR TITLE
Bug 1177432 - [Settings] Introduce gaia-checkbox in developer panel

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1177432

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d05ddc2a03f32d9d5672e6eaf5bf5ede9fed5674

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=92fdc53443e0cba92413e9a9377ddce10a22a994

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=63a29f983d30d3ca9bbd02ec88433b7d2198920d

File: shared/elements/gaia_checkbox/script.js [hunk 1]
```diff
@@ -109,6 +108,18 @@ window.GaiaCheckbox = (function(win) {
     }
   });
 
+  /**
+   * Proxy the name property to the input element.
+   */
+  Object.defineProperty( proto, 'name', {
```
Comment (crh0716) [inline shared/elements/gaia_checkbox/script.js]: nit: redundant space

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2ea8ed556bc984d40c6bc816bba5026a820c1d9d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6ca15a17322aae55f9022194357a1399d240d5b5

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f11d40c637dd70382f87ef0c3d05879e4a5c90d9

**Pull request merged**